htslib/ksort.h

```diff
@@ -88,7 +88,11 @@ typedef struct {
 	int depth;
 } ks_isort_stack_t;
 
+#ifndef __cplusplus
```

Comment:
You could probably just get rid of the `register` annotation.  I doubt that it makes much difference for modern compilers.